gspread/spreadsheet.py

```diff
@@ -518,12 +518,26 @@ def share(
             with_link=with_link,
         )
 
-    def export_to_pdf(self):
-        """Export the spreadsheet as a pdf.
+    def export(self, format):
```

Comment:
please set the default value for `format` to `ExportFormat.PDF`, this is what was requested at first by the user and will be most likely the requested format.